src/drivers/code_hosting/github.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local branch=$2
+
+  open_browser "https://github.com/$repository/compare/$branch?expand=1"
```

Comment:
Plus, I'd like to keep the responsibility of creating the pull request in the driver. For example, if we add a "hub" driver here, we could call `hub` here to create the PR for us. If we limit the code hosting driver family to ever only open browsers to create new PRs, we would limit ourselves here unnecessarily.